src/backend/core/cart/models.py

```diff
@@ -66,6 +66,20 @@ def __str__(self) -> str:
             f"{self.shipping_method} - {self.country} - {self.vat_group} - {self.price}"
         )
 
+    @property
+    def price_incl_vat(self):
+        if not self.vat_group:
+            return self.price
+        return self.price * (1 + self.vat_group.rate / 100)
+
+    @property
+    def formatted_price_incl_vat(self):
+        price_str = f"{self.price_incl_vat:.2f}".replace(".", ",")
```

Comment:
You ran into similar problem as me 🙂 
I also needed to implement format price methods for cart item and cart

However, I'd be careful about the `,`, because different languages use [different decimal separators](https://en.wikipedia.org/wiki/Decimal_separator#Conventions_worldwide)

IMO, we have 2 options:
- don't care about these separators at all and simply hardcode it. But we should be at least consistent (I think we use dots for everything now, except this 😄 )
- add decimal_separator as a field to country

Reply:
> You ran into similar problem as me 🙂
I also needed to implement format price methods for cart item and cart

Haha, so we're about to have a conflict? 😄 

> However, I'd be careful about the ,, because different languages use [different decimal separators](https://en.wikipedia.org/wiki/Decimal_separator#Conventions_worldwide) 
That's a very good point, I did not even think about that. I think the easiest would be to handle it on `Currency` model, considering it's `language` bias we might actually store it in the `Country` model. So new issue? 😊

Reply:
> Haha, so we're about to have a conflict?

Fortunately not, because I updated different models - Cart and CartItem

Reply:
We can definitely put this into a new issue, if we want to solve this problem.
I'm also not sure whether we want this price formatting for storefrontly only, or for dashboard as well (since dashboard isn't localized, we may simply leave the dashboard as it is 🙂)

Reply:
>  if we want to solve this problem. 

So maybe #nicetohave and let's keep it with a dot? I wouldn't care much about dashboard in this way honestly